### ISSUE TITLE
QST: Question regarding documentation 

### ISSUE THREAD

**Issue by borisRa** [deepchecks/deepchecks#988]
### Research

- [X] I have searched the [[deepchecks] tag](https://stackoverflow.com/questions/tagged/deepchecks) on StackOverflow for similar questions.

- [X] I have asked my usage related question on [StackOverflow](https://stackoverflow.com).

### Link to question on StackOverflow

https://docs.deepchecks.com/en/stable/user-guide/when_should_you_use.html#when-should-you-use-new-data

### Question about deepchecks

Hi,
Based on your documentation page => https://docs.deepchecks.com/en/stable/user-guide/when_should_you_use.html#when-should-you-use-new-data

There are 4 phases :
1)New Data: Single Dataset Validation -> For these purposes you can use the single_dataset_integrity()
2)After Splitting the Data: Train-Test Validation   -> For these purposes you can use the -> train_test_validation()
3)After Training a Model: Analysis & Validation ->model_evaluation()
4)General Overview: Full Suite -> full_suite()

The question where can I see the examples of using this 4 options ?

Thanks,
Boris

**Comment by borisRa** [comment]
Thanks !
> 2\. The train_test_validation() -> in the usecase example this function is used with a pre-trained model. why ?
vsuite = train_test_validation()
vsuite.run(model=logreg, train_dataset=ds_train, test_dataset=ds_test)

I mean from the explanation above , model_evaluation() should use model and   train_test_validation() just validating the split distributions .

Another question :
How can I export the generated report to an HTML file instead of working with a Notebook ?

Thanks !
Boris

**Comment by noamzbr** [comment]
> 2)After Splitting the Data: Train-Test Validation -> For these purposes you can use the -> train_test_validation()

You are right, for the checks in the train_test_validation() suite it is not necessary to pass the model, but if passed the model will be used to calculate feature importance for prioritization of different features in the check displays.

**Comment by borisRa** [comment]
> suite

Thanks !
Tried this and then wanted to see the results through a HTML file .
Got the error bellow .

My code :
`	from deepchecks.suites import train_test_validation

	vsuite = train_test_validation()

	vsuite.run( train_dataset=df_full, test_dataset=df_full)

	vsuite.save_as_html('my_suite.html')`

and the error is :
`AttributeError: 'Suite' object has no attribute 'save_as_html'`

Used this page as an example :  https://docs.deepchecks.com/en/stable/examples/guides/save_suite_result_as_html.html

How can I fix this ?

Thanks !
Boris

**Comment by matanper** [comment]
@borisRa Probably you are on an older version of `deepchecks`, can you try to update (`pip install -U deepchecks`) and see if that works?

**Comment by borisRa** [comment]
> pip install -U deepchecks

ver 0.5.0 ( the last one from pypi)

**Comment by matanper** [comment]
@borisRa Sorry I haven't looked at your code properly! `suite.run` method returns a `SuiteResult` object. you should run it like this:
```
vsuite = train_test_validation()

result = vsuite.run( train_dataset=df_full, test_dataset=df_full)

result.save_as_html('my_suite.html')`
```

**Comment by borisRa** [comment]
> @borisRa Sorry I haven't looked at your code properly! `suite.run` method returns a `SuiteResult` object. you should run it like this:
> 
> ```
> vsuite = train_test_validation()
> 
> result = vsuite.run( train_dataset=df_full, test_dataset=df_full)
> 
> result.save_as_html('my_suite.html')`
> ```

Thanks this worked !

Now getting a HTML without any plots to compare distributions . **why ?**
This is what I get :

![image](https://user-images.githubusercontent.com/3207061/158068169-b74d4786-ce42-4e52-8058-dfa782548ee1.png)

**Comment by matanper** [comment]
@borisRa Do you see there are tabs? Please have a look in all 3 of them

**Comment by borisRa** [comment]
> @borisRa Do you see there are tabs? Please have a look in all 3 of them

Thanks !
Found the problem. can't use pandas data frame as input to train_test_validation() must use deepchecks's built-in Dataset.
